### PR TITLE
[ADD] init hook to set gender based on titles

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: partner_contact_gender/tests/test_partner_contact_gender.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
+
+
+class TestPartnerContactGender(TransactionCase):
+    def test_partner_contact_gender(self):
+        from ..hooks import post_init_hook
```
Comment (pedrobaeza) [inline partner_contact_gender/tests/test_partner_contact_gender.py:16]: You should put in the setUp a test partner with any of the titles, but without gender, call the hook, and then assert the gender to actually test something.

File: partner_contact_gender/__init__.py [hunk 2]
```diff
@@ -17,3 +17,4 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from . import models
+from hooks import post_init_hook
```
Comment (pedrobaeza) [inline partner_contact_gender/__init__.py]: `from .hooks import post_init_hook`

**Comment by hbrunn** [comment]
@pedrobaeza thanks, done

**Comment by pedrobaeza** [comment]
:+1: 

Thanks

**Comment by gurneyalex** [comment]
:+1:

**Pull request merged**